public/js/helpers/add_day.js

```diff
@@ -0,0 +1,9 @@
+var addDay = function (dataSet) {
+  console.log(addDay);
+  var plusDay = parseInt(dataSet.date.split('-')[2]) + 1;
+  var plusDate = dataSet.date.split('-');
+  plusDate.pop();
+  plusDate.push(plusDay.toString());
+  const actualDateTo = new Date(plusDate[0], plusDate[1], plusDate[2]);
```

Comment:
this and the previous 4 lines can be condensed, e.g.
```js
var splitDate = date.split('-')
var newDate = new Date(splitDate[0], splitDate[1], splitDate[2] + 1)
```

Reply:
i tested that +1 it doesn't work